File: CS/ScaleIntegerSpacing/SalesPersonData.cs
using System;
using System.Collections.Generic;

namespace ScaleIntegerSpacing {
    public class ViewModel {
        public DateTime Date { get; set; }
        public double Value { get; set; }

        public static List<ViewModel> CreateData() {
            List<ViewModel> data = new List<ViewModel>();
            var rnd = new Random(100);
            for(int i = 0; i < 12; i++) {
                ViewModel record = new ViewModel();
                record.Date = DateTime.Now.AddDays(i);
                record.Value = rnd.Next(0, 5);
                data.Add(record);
            }
            return data;
        }
    }
}

File: VB/ScaleIntegerSpacing/SalesPersonData.vb
Imports System
Imports System.Collections.Generic

Namespace ScaleIntegerSpacing
	Public Class ViewModel
		Public Property [Date]() As Date
		Public Property Value() As Double

		Public Shared Function CreateData() As List(Of ViewModel)
			Dim data As New List(Of ViewModel)()
			Dim rnd = New Random(100)
			For i As Integer = 0 To 11
				Dim record As New ViewModel()
				record.Date = Date.Now.AddDays(i)
				record.Value = rnd.Next(0, 5)
				data.Add(record)
			Next i
			Return data
		End Function
	End Class
End Namespace